src/main/java/com/example/ShortnerURL/Controller/PingController.java

```diff
@@ -15,16 +17,18 @@ public class PingController {
     private JdbcTemplate jdbcTemplate;
 
     @GetMapping("/ping")
-    public ResponseEntity<Map<String, String>> ping() {
+    public ResponseEntity<Object> ping() {
```

Comment:
using `Object` here removes type safety, and allows you to return any object. I see why you ended up doing this, but this is not the solution to your problem.

Reply:
once errors go via `@ExceptionHandler` then this would be
```java
public SuccessResponseDto ping() {
```